### PR TITLE
Add get-predeployed-accounts test

### PR DESCRIPTION
**Pull request by qperrot**
Add get-predeployed-accounts.test.ts, which test the starknet.devnet.getPredeployedAccounts() function. This function fetch pre-deployed accounts and return an array, containing the data of all pre-deployed accounts (which are: address, initial_balance, private and public keys).

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -8,12 +8,12 @@ import "@nomiclabs/hardhat-ethers";
 const config: HardhatUserConfig = {
   solidity: '0.6.12',
   starknet: {
-    dockerizedVersion: "0.8.2", // alternatively choose one of the two venv options below
+    // dockerizedVersion: "0.8.2", // alternatively choose one of the two venv options below
```
Comment (FabijanC) [inline hardhat.config.ts:11]: The changes done in this file aren't needed for the tests to work. You should revert them.

File: test/get-predeployed-accounts.test.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    it("should pass", async () => {
+        const response = await starknet.devnet.getPredeployedAccounts();
+        expect(response).to.be.an('array')
+        expect(response[0].address).to.be.a('string')
+        expect(response[0].initial_balance).to.be.a('number')
+        expect(response[0].private_key).to.be.a('string')
+        expect(response[0].public_key).to.be.a('string')
+    });
+})
```
Comment (FabijanC) [inline test/get-predeployed-accounts.test.ts:15]: GitHub is reporting a missing newline here.

File: test/get-predeployed-accounts.test.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { expect } from "chai";
+import { starknet } from "hardhat"
+
+describe("Devnet restart", function() {
+    this.timeout(900_000);
```
Comment (FabijanC) [inline test/get-predeployed-accounts.test.ts:5]: See how timeout is specified in other tests.

**Review by FabijanC — CHANGES_REQUESTED** [review]
Edit the PR description to give a short summary of what you are introducing with this PR.

Also, some of the tests on your plugin PR are failing on CircleCI because this repo's `predeployed-accounts-test` branch was not created off the top of the `plugin` branch. To fix this, you can do `git rebase plugin` while on `predeployed-accounts-test`.

File: test/get-predeployed-accounts.test.ts (hunk 2 above)
Comment (qperrot) [reply test/get-predeployed-accounts.test.ts]: I add a new line at the end [48fde15](https://github.com/Shard-Labs/starknet-hardhat-example/pull/66/commits/48fde15de89778456fb896baec7b05938e12a5b2)

File: test/get-predeployed-accounts.test.ts (hunk 3 above)
Comment (qperrot) [reply test/get-predeployed-accounts.test.ts]: I import TIMEOUT from constant and replace 900_000 with it [48fde15](https://github.com/Shard-Labs/starknet-hardhat-example/pull/66/commits/48fde15de89778456fb896baec7b05938e12a5b2)

**Pull request merged**